<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EEGVIz</class>
 <widget class="QMainWindow" name="EEGVIz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1711</width>
    <height>1006</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout" columnstretch="10,1,1">
      <item row="1" column="0" rowspan="3">
       <widget class="QWidget" name="widget" native="true">
        <property name="font">
         <font>
          <family>Arial</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(182, 182, 182);</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="1">
          <widget class="QScrollBar" name="waveform_time_scroll_bar">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QScrollBar" name="channel_scroll_bar">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QGroupBox" name="edf_info">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>13</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="title">
         <string>EDF Information</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <layout class="QGridLayout" name="gridLayout_18">
         <item row="3" column="0">
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>11</pointsize>
             <weight>50</weight>
             <bold>false</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Length:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>11</pointsize>
             <weight>50</weight>
             <bold>false</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Sampling Frequency:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="sampfreq">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>11</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="numchannels">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>11</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="filename">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>11</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>11</pointsize>
             <weight>50</weight>
             <bold>false</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Number of Channels:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="length">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>11</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>11</pointsize>
             <weight>50</weight>
             <bold>false</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>File Name:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="title">
             <string>Display</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Number of Channels to Display</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="n_channel_input">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="font">
                <font>
                 <family>Arial</family>
                </font>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_11">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Display Time Window</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="display_time_window_input">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Sec</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_14">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Display Time Window Increment</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="Time_Increment_Input">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_22">
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>%</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QCheckBox" name="normalize_vertical_input">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Normalize Vertically</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="plot_time_frequency_button">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Show TF plot of specific channel</string>
        </property>
        <property name="text">
         <string>Plot Time Frequency</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="Choose_Channels_Button">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Choose specific channels to display</string>
          </property>
          <property name="text">
           <string>Choose Channels</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="waveform_plot_button">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Refresh the plot with changes</string>
          </property>
          <property name="text">
           <string>Update Plot</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_EDF_toolbar"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Open_EDF">
   <property name="text">
    <string>&amp;Open EDF</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Load_Detection">
   <property name="text">
    <string>&amp;Load Detection</string>
   </property>
  </action>
  <action name="actionQuick_Detection">
   <property name="text">
    <string>&amp;Quick Detection</string>
   </property>
  </action>
  <action name="action_saveAs_CSV">
   <property name="text">
    <string>saveAs_CSV</string>
   </property>
   <property name="toolTip">
    <string>save file as CSV</string>
   </property>
  </action>
  <action name="action_saveAs_PKL">
   <property name="text">
    <string>saveAs_PKL</string>
   </property>
   <property name="toolTip">
    <string>save file as PKL</string>
   </property>
  </action>
  <action name="actionOpen_EDF_toolbar">
   <property name="text">
    <string>Open EDF</string>
   </property>
   <property name="toolTip">
    <string>Open EDF</string>
   </property>
  </action>
  <action name="actionLoad_Detection_toolbar">
   <property name="text">
    <string>Load Detection</string>
   </property>
   <property name="toolTip">
    <string>Load Detection</string>
   </property>
  </action>
  <action name="actionQuick_Detection_toolbar">
   <property name="text">
    <string>Quick Detection</string>
   </property>
   <property name="toolTip">
    <string>Quick Detection</string>
   </property>
  </action>
  <action name="action_filter">
   <property name="text">
    <string>filter</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
